### PR TITLE
docs: add information on use with fastify

### PR DESCRIPTION
**Pull request by pearofducks**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Document use (for now) with Fastify while #617 is worked on - addresses concerns raised in #602 .

This also can serve as a location in the README for documentation on other servers once #617  is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -340,6 +342,32 @@ a modification, please feel free to create an issue on Github. _Note: The issue
 template isn't optional, so please be sure not to remove it, and please fill it
 out completely._
 
+## Other servers
+
+While Express is the primary target of this middleware, support for other servers [is in-progress](https://github.com/webpack/webpack-dev-middleware/pull/617).
```
Comment (hiroppy) [inline README.md:347]: I don't think we need this text so I want to merge that PR at first.

Comment (pearofducks) [reply README.md]: Removed!

File: README.md [hunk 2]
```diff
@@ -340,6 +342,30 @@ a modification, please feel free to create an issue on Github. _Note: The issue
...
+
+### Fastify
+
+Fastify interop will require the use of `fastify-express` instead of `middie` for providing middleware support. As the authors of `fastify-express` recommend, this should only be used as a stopgap while full Fastify support is worked on.
+
+```js
+const fastify = require('fastify')()
+const webpack = require('webpack')
+const webpackConfig = require('./webpack.config.js')
+const devMiddleware = require('webpack-dev-middleware').default
```
Comment (evilebottnawi) [inline README.md:357]: we fixed it :+1: no need `default`

**Review by evilebottnawi — COMMENTED** [review]
Thanks for help, let's fix one thing

**Comment by evilebottnawi** [comment]
Let's merge, we should do release ASAP, because we need many work on webpack-dev-server side too

**Pull request merged**